src/targets/flysky.ini

```diff
@@ -0,0 +1,29 @@
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:FRM301_TX_via_STLINK]
+extends = env_common_stm32, radio_2400
+board = FRM301
+debug_tool = stlink
+build_flags =
+	${env_common_stm32.build_flags}
+	${common_env_data.build_flags_tx}
+	${radio_2400.build_flags}
+	-include target/FRM301_TX.h
+	-DHSE_VALUE=12000000U
+	#-DVECT_TAB_OFFSET=0x1000U
```

Comment:
Bootloader won't be used?

Reply:
Sorry, i couldn't work on it the past days.  Yes, if there is enough flash i wanted to get the bootloader too but i wanted to first get the firmware working properly.